src/components/button/index.ts

```diff
@@ -0,0 +1,3 @@
+export const button = (a: string) => {
```

Comment:
Remind me to remove this file after review is done. It is for testing/review/demo purposes

Reply:
`npm run type`

- Pass
- Remove `: string`
- error TS7006: Parameter 'a' implicitly has an 'any' type

👍

Reply:
I guess I will say I feel fine about you removing this now, unless someone else wants to run it, break it, and run it again.

Reply:
Obviously, it also passed here:

![image](https://user-images.githubusercontent.com/771447/80771743-39dd6a80-8b09-11ea-8995-6a93bd980ec9.png)

Reply:
Reminder to remove this file after the review is done. 🙂